services/fhir.ts

```diff
@@ -276,3 +376,31 @@ export function getConcepts(valueSetId: string, params?: SearchParams): Promise<
         params: { ...params },
     });
 }
+
+export async function applyFHIRService<T, F>(request: AxiosRequestConfig): Promise<RemoteDataResult<T, F>> {
+    return service(request);
+}
+
+export async function applyFHIRServices<R extends AidboxResource, T, F>(
+    requests: Array<AxiosRequestConfig>,
+    type: 'transaction' | 'batch' = 'transaction'
+): Promise<RemoteDataResult<T[], F[]>> {
+    return service({
+        method: 'POST',
+        url: '/',
+        data: {
+            type,
+            entry: requests.map(({ method, url, data, headers }) => {
+                return {
+                    ...(data ? { resource: data } : {}),
+                    request: {
+                        method,
+                        url,
+                        ...(headers && headers['If-Match'] ? { ifMatch: headers['If-Match'] } : {}),
```

Comment:
It's OK that we support If-None-Exist. I prefer moving this mapping AxiosRequestConfig -> BundleEntry into a separate function. It'll be easier to cover it with unit tests. Also we can add support for If-Modified-Since, If-None-Match headers. For some future features it might be useful.